nmostesting/GenericTest.py

```diff
@@ -352,8 +352,8 @@ def check_response(self, schema, method, response):
 
         try:
             self.validate_schema(response.json(), schema)
-        except jsonschema.ValidationError:
-            return False, "Response schema validation error"
+        except jsonschema.ValidationError as e:
+            return False, f"Response schema validation error. Hint: {e.message}"
```

Comment:
Are we allowing ourselves f-strings?

Reply:
It was introduced in 3.6 and the docs require 3.9 so hopefully yes?